libs/sdk-core/src/breez_services.rs

```diff
@@ -737,6 +737,12 @@ impl BreezServices {
         Ok(None)
     }
 
+    pub async fn rescan_swaps(&self) -> SdkResult<()> {
```

Comment:
This could return the list of updated swaps, to spare the caller another `in_progress_swap`.

Also: does this mean `in_progress_swap` can now return a list? If I understand this correctly, it can now be that any number of previously-closed swaps can become in-progress if they receive sats on those addresses.

Reply:
If we want to return something here then we need also to include the refundables (which may also be updated)
I think since the user has the API to retrieve them we can return nothing as you suggested

Regarding in progress if they get funds after the redeem they become refundables so it should not be a problem